mail-filter/dcc/dcc-1.3.158.ebuild

```diff
@@ -63,27 +75,29 @@ src_configure() {
 moveconf() {
 	local into=/etc/dcc/
 	for i in $@; do
-		mv "${D}${dcc_homedir}/${i}" "${D}${into}"
-		dosym "${into}${i}" "${dcc_homedir}/${i}"
+		mv "${D}/${dcc_homedir}/${i}" "${D}/${into}"
```

Comment:
missing `|| die`

Reply:
You're adding `EPREFIX` everywhere, but left `${D}` everywhere?

Reply:
Done